BenchBnb/BenchBnb/Views/Register/Index.cshtml

```diff
@@ -0,0 +1,7 @@
+﻿
+@{
+    ViewBag.Title = "Index";
+}
+
+<h2>Register Index</h2>
```

Comment:
I'd remove this view and the associated action method in the `RegisterController` class.

Reply:
Fixed